packages/babel-core/test/fixtures/transformation/source-maps/input-source-map-multiple-output-sources-complete-replace/source-map.json

```diff
@@ -1,15 +1,13 @@
 {
   "version": 3,
-  "mappings": "AAAC,KCAD;ACAA,K",
+  "mappings": "AAAC,KAAG;ACAJ,KAAG",
```

Comment:
This change is fine. The `;` is mapped to the `bar.js` file (which fully replaces the `input.tsx` in the plugin) instead of the `input.tsx` file. Because nothing refers to the `input.tsx` file, it's removed from `sourcesContent`.